.codacy.yml

```diff
@@ -1,2 +1,3 @@
 ---
-exclude_paths: []
+exclude_paths:
+  - ".github/styles/**"
```

Comment:
I think this change is working correctly as expected, as Codacy is no longer reporting issues inside the `styles/` directory. :+1: